tests/conftest.py

```diff
@@ -1145,6 +1145,17 @@ def reset_logging_config():
     logging.config.dictConfig(logging_config)
 
 
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
```suggestion
@pytest.fixture(scope="module", autouse=True)
```

Suggestion, to do it once per module. We potentially overload logger during the tests runs, at least a saw side effects of that but can't find where it happen.

Reply:
Why not.